src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -99,5 +109,13 @@ private IEnumerable<ContentTypeDefinition> GetContainedContentTypes(BuildPartDis
             var contentTypes = settings.ContainedContentTypes ?? Enumerable.Empty<string>();
             return contentTypes.Select(contentType => _contentDefinitionManager.GetTypeDefinition(contentType));
         }
+
+        private async Task<bool> IsContentCreatableAsync(ContentTypeDefinition contentTypeDefinition)
```

Comment:
"IsAuthorizedToCreate" would be a more fitting neme here and for the property too.